scripts/run-all-migrations.sh

```
@@ -0,0 +1,237 @@
+      echo "  --stop-on-error  Para na primeira falha"
+      echo "  --verbose        Mostra output completo do psql"
+      exit 0
+      ;;
+  esac
+done
+
+# Verificar DATABASE_URL
+if [ -z "${DATABASE_URL:-}" ]; then
+  # Tentar carregar do .env
+  if [ -f "${SCRIPTS_DIR}/../.env" ]; then
+    export $(grep -E '^DATABASE_URL=' "${SCRIPTS_DIR}/../.env" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)